content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -342,6 +342,10 @@ rules:
 The following examples are excerpts from Role or ClusterRole objects, showing only
 the `rules` section.
 
+Note: When specifying resources in a ClusterRole or Role, always use the plural and lowercase form of the resource name.
+For example, use pods instead of pod, services instead of service.
+Incorrect resource names will cause RBAC policies to fail.
```

Comment:
The important part is that it match the REST API URL, which for built-in types is always plural and lowercase, but might not be for aggregated API servers.

```suggestion
Note: When specifying resources in a ClusterRole or Role, always use the resource exactly as it appears in the REST API path segment.
For example, use `pods` instead of `pod` or `Pods`.
Resources that do not exactly match the REST API path segment will not authorize the request.
```